css/quotes.css

```diff
@@ -1,4 +1,180 @@
-/* Quote page specific styles */
+* {
+    box-sizing: border-box;
+}
 
+body {
+    margin: 0;
+}
 
-/* We should use a mobile first approach having out breack point at 768px */
\ No newline at end of file
+.form {
+    margin-left: 5%;
+    margin-right: 5%;
+    max-width: 90%;
+}
+
+.form_title {
```

Comment:
Composed words should go with middle dash `.homepage-hero` and element children should go with a double low dash `.form__title`

Reference: http://getbem.com/introduction/